tests/src/test/scala/org/apache/openwhisk/core/limits/ActionLimitsTests.scala

```diff
@@ -463,4 +463,34 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers with WskActorSys
       _.response.result.get.fields("error") shouldBe Messages.memoryExhausted.toJson
     }
   }
+
+  /**
+   * Test that a heavy logging action is interrupted within its timeout limits.
+   */
+  it should "interrupt the heavy logging action within its time limits" in withAssetCleaner(wskprops) {
+    (wp, assetHelper) =>
+      val name = "NodeJsTestLoggingActionCausingTimeout-" + System.currentTimeMillis()
+      print(s"\n create action ${name} using api host: ${wskprops.apihost}..")
+      assetHelper.withCleaner(wsk.action, name, confirmDelete = true) { (action, _) =>
+        action.create(
+          name,
+          Some(TestUtils.getTestActionFilename("loggingTimeout.js")),
+          timeout = Some(allowedActionDuration),
+          kind = Some("nodejs:8"))
+      }
```

Comment:
Travis output indicates that the test action could not be created:

```
org.apache.openwhisk.core.limits.ActionLimitsTests > Action limits should interrupt the heavy logging action within its time limits STANDARD_OUT
     create action NodeJsTestLoggingActionCausingTimeout-1550822594899 using api host: 172.17.0.1..Exception occurred during test execution: org.scalatest.exceptions.TestFailedException: 400 was not equal to 200
```

Reply:
Does the kind have to nodejs 8? Is this because the default kind is 6 as of now?

Reply:
due to the limited runtime support by the system tests this error occurred, we exchanged the action with a plain nodejs:6 version